React/src/components/Auth/index.tsx

```diff
@@ -0,0 +1,14 @@
+import LogoWolox from 'resources/img/logoWolox.png';
+
+import styles from './styles.module.scss';
+
+export function AuthContainer({ children }: any) {
+  return (
+    <div className={styles.pageContainer}>
+      <div className={styles.loginContainer}>
```

Comment:
This won't be just for the login.
```suggestion
      <div className={styles.authContainer}>
```